src/fields/DateField.jsx

```diff
@@ -1,12 +1,61 @@
 import React from 'react'
+import Datepicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+import { format } from 'date-fns'
+import { fr, enUS } from 'date-fns/locale'
 
 import { block } from '../utils'
-import InputField from './InputField'
+
+import './Datefield.css'
 
 @block
 export default class DateField extends React.PureComponent {
-  render(b) {
-    const { className, ...props } = this.props
-    return <InputField className={b.mix(className)} {...props} />
+  componentDidMount() {
+    const { readOnly, disabled, elementRef } = this.props
+    if (!(readOnly || disabled)) {
+      elementRef.current = this.datepicker.input
+    }
+  }
+  componentWillUnmount() {
+    const { elementRef } = this.props
+    elementRef.current = null
+  }
+  render() {
+    const {
+      value,
+      onBlur,
+      onChange,
+      onFocus,
+      i18n,
+      readOnly,
+      required,
+      name,
+      placeholder,
+    } = this.props
+    const restOfProps = { onBlur, onFocus, readOnly, required, name }
+    const isLocaleFr = i18n.calendar.locale === 'fr'
+    const locale = isLocaleFr ? fr : enUS
+    const dateFormat = isLocaleFr ? 'dd/MM/yyyy' : 'MM/dd/yyyy'
+
+    return (
+      <Datepicker
+        ref={ref => (this.datepicker = ref)}
+        selected={value ? new Date(value) : null}
+        onChange={date => {
+          this.setState({ date })
+          onChange(date ? format(date, 'yyyy-MM-dd') : '')
+        }}
+        isClearable
+        locale={locale}
+        dateFormat={dateFormat}
+        className="Formol_InputField Formol_DateField Formol_Field__element"
```

Comment:
`formol` uses `bemboo` for class names management, it should be good to use it here too (via the `@block` decorator).

Reply:
Fait dans f88852f